vcd/datasource_vcd_catalog_media.go

```diff
@@ -0,0 +1,76 @@
+package vcd
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func datasourceVcdCatalogMedia() *schema.Resource {
+	return &schema.Resource{
+		Read: resourceVcdMediaRead,
+
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"catalog": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "catalog name where upload the Media file",
+			},
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "media name",
+			},
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"metadata": {
+				Type:        schema.TypeMap,
+				Computed:    true,
+				Description: "Key and value pairs for catalog item metadata",
+				// For now underlying go-vcloud-director repo only supports
+				// a value of type String in this map.
+			},
+			"is_iso": &schema.Schema{
```

Comment:
This question should have been asked when the resource was created, but let's see it here:
Why properties `is_iso` and `is_published` are strings instead of Boolean, and why is `size` a string instead of a number?

Reply:
Cause they are computed - so no difference, but I look if there is other implications